src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
>I was thinking of this
>
>```rust
pub trait RgbStandard {
    type Primaries: Primaries;
    type WhitePoint: WhitePoint;
    type TransferFn: TransferFn;
}
```

That's essentially the same thing as what we already have. `Primaries` and `WhitePoint` are currently just grouped.

>The white point of the primaries have no relation to the color space white point. The primaries trait needs its own white point 

Does the white point even matter at all, in this case? I mean, yxY values are the same anyway, right? Or does sRGB + D50 imply that _the whole_ space is transformed into the D50 space, instead of just changing the white point? The details are not really clear to me...